akka-remote/src/test/serializater-testkit/17/a/Quarantined/2.5.0

```diff
@@ -0,0 +1,6 @@
+
+
+
+abcdefhost �`�
+
+abcdefhost �`�
```

Comment:
Looks like git isn't correctly detecting these as binary files, so it might be a good idea to explicitly add attributes: https://git-scm.com/book/en/v2/Customizing-Git-Git-Attributes

This can help avoid issues on Windows with automatic line ending translation.